packaging/technical-addon/Splunk_TA_otel/windows_x86_64/bin/Splunk_TA_otel_utils.ps1

```diff
@@ -60,7 +60,7 @@ try {
 	$otelProcess = Get-Win32_ProcessByNameAndParentId `
 		$otelProcessName `
 		$parentProcess.ProcessId `
-		-timeout ([TimeSpan]::FromSeconds(5)) `
+		-timeout ([TimeSpan]::FromSeconds(10)) `
```

Comment:
Is this related to the rest of the PR, or just addressing some flakiness while we're here?